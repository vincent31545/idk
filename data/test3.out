First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Louis Rubin Memorial Aproach]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Panera Bread]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Louis Rubin Memorial Aproach]
Unknown building: [Panera Bread]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [200]
Unknown building: [222]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [201]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [333]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from OGE to Academy Hall:
	Walk S to (Intersection 108)
	Walk S to (Intersection 106)
	Walk S to (Intersection 140)
	Walk S to (Intersection 139)
	Walk NW to (CBIS)
	Walk S to (Academy Hall)
Total distance: 848.447 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [92]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [95]
Unknown building: [97]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [98]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [99]
