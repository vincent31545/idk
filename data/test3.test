r
Louis Rubin Memorial Aproach
67
r
76
Panera Bread
r
Louis Rubin Memorial Aproach
Panera Bread
r
200
222
r
74
201
r
333
23
r
91
67
r
76
92
r
95
97
r
98
EMPAC
r
Polytechnic Residence Commons
99
q

