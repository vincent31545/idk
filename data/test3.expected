First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Louis Rubin Memorial Aproach]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Panera Bread]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [Louis Rubin Memorial Aproach]
Unknown building: [Panera Bread]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [200]
Unknown building: [222]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [201]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [333]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from OGE to Academy Hall:
	Walk S to (Intersection 108)
	Walk S to (Intersection 106)
	Walk S to (Intersection 140)
	Walk S to (Intersection 139)
	Walk S to (Intersection 102)
	Walk W to (Academy Hall)
Total distance: 784.958 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [92]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [95]
Unknown building: [97]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [98]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [99]

